services/venice-controller/src/main/java/com/linkedin/venice/controller/server/SchemaRoutes.java

```diff
@@ -177,6 +169,34 @@ public Route addDerivedSchema(Admin admin) {
     };
   }
 
+  public Route updateValueSchema(Admin admin) {
+    return (request, response) -> {
+      SchemaResponse responseObject = new SchemaResponse();
+      response.type(HttpConstants.JSON);
+      try {
+        AdminSparkServer.validateParams(request, UPDATE_VALUE_SCHEMA.getParams(), admin);
```

Comment:
Before validating params, let's add a check to ensure that only allowlist users could run this command.

Reply:
Thanks for your input, I've updated my code with those changes :)